prow/sinker/deployment.yaml

```diff
@@ -9,6 +9,11 @@ metadata:
     wave.pusher.com/update-on-config-change: "true"  
 spec:
   replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
```

Comment:
Any particular reason to override the defaults? What happens if there is > 1 of these?

Reply:
This was an assumption based on a lack of a don't-scale comment. No longer relevant.